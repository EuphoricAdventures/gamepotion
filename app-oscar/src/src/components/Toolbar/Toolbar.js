import React from 'react'
import styled from 'styled-components'

const StyledToolbar = styled.ul`
  display: flex;
  flex-direction: row;
  list-style-type: none;
  background-color: #2e3131;
`

const Toolbar = ({ children }) => (
  <StyledToolbar>
    {children}
  </StyledToolbar>
)

export default Toolbar
