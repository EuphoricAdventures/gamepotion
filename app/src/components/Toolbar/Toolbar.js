import React from 'react'
import PropTypes from 'prop-types'
import ToolbarButton from '../ToolbarButton/ToolbarButton'
import styled from 'styled-components'

const StyledToolbar = styled.ul`
  display: block;
  height: 48px;
  background-color: rgb(64, 64, 64);
`

const Toolbar = ({ children, href }) => (
  <StyledToolbar href={href}>
    {children}
  </StyledToolbar>
)

Toolbar.propTypes = {
}

Toolbar.defaultProps = {
}

export default Toolbar
