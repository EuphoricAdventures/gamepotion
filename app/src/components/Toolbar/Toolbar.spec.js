import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Toolbar from './Toolbar'

it('renders', () => {
  const tree = renderer
    .create(<Toolbar></Toolbar>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
