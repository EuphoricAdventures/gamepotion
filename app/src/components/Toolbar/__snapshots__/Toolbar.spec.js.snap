// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  list-style-type: none;
  background-color: #2e3131;
  overflow: scroll;
  box-shadow: 0 2px 4px rgb(172,172,172);
}

<ul
  className="c0"
/>
`;
